footylib/footylib.py

```diff
@@ -112,7 +112,7 @@ def teams(self):
                                       {'class': 'leaguemanager standingstable'})
             for teams in standings:
                 for row in teams.find_all('tr', {'class': ('alternate', '')}):
-                    self._teams.append(Team(row))
+                    self._teams.append(Team(self, self.url, row))
```

Comment:
Maybe you don't need to pass self.url here, since it is part of the self you pass right before? Do you think that it helps if it is so explicit?